### PR TITLE
mkck: use minimal images

### PR DESCRIPTION
**Pull request by kshtsk**
Signed-off-by: Kyr Shatskyy <kyrylo.shatskyy@suse.com>

### CODE REVIEW

File: conf/makecheck-sle-15.1.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 flavor: b2-30
-image: makecheck-sle-15.1-x86_64
+#image: makecheck-sle-15.1-x86_64
+image: minimal-sle-15-sp1.x86_64-openstack
```
Comment (toabctl) [inline conf/makecheck-sle-15.1.yaml:3]: Wrong name after https://gitlab.suse.de/ceph/storage-ci/-/merge_requests/116 gets merged.

File: conf/makecheck-sle-15.2.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+flavor: b2-30
+#image: makecheck-sle-15.1-x86_64
+image: minimal-sle-15-sp2.x86_64-openstack
```
Comment (toabctl) [inline conf/makecheck-sle-15.2.yaml:3]: same here - wrong name

File: conf/makecheck-sle-15.1.yaml (hunk 1 above)
Comment (kshtsk) [reply conf/makecheck-sle-15.1.yaml:3]: That's fine, feel free to commit corresponding change, I was working with the images I had.

**Review by toabctl — COMMENTED** [review]
@kshtsk updated image names is here: https://github.com/SUSE/sesci/pull/24

**Pull request merged**